package.json

```diff
@@ -49,6 +49,7 @@
     "pg": "8.7.1",
     "plotly.js-basic-dist-min": "1.57.1",
     "react": "16.8.6",
+    "react-contexify": "^6.0.0",
```

Comment:
Moving the implementation of creating context menu in main process, I added `react-contextify` instead.